CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # bedrock-vue-wizard ChangeLog
 
+## 3.0.0 - TBD
```

Comment:
are you sure this is 3.0.0?

Reply:
I have no idea. I assumed since this was a large upgrade we will go up to next version but I have no clue if that is correct or not.

Reply:
I think you are correct, just not entirely 100% sure.